mock/src/main/java/com/tngtech/keycloakmock/impl/handler/TokenRoute.java

```diff
@@ -145,6 +145,7 @@ private void handleClientCredentialsFlow(final RoutingContext routingContext) {
   private String toTokenResponse(String token, String sessionId) {
     return new JsonObject()
         .put("access_token", token)
+        .put("token_type","Bearer")
```

Comment:
It seems the Google Java formatter complains. I assume it should be as simple as this:
```suggestion
        .put("token_type", "Bearer")
```

But you can also install pre-commit and run it to make sure.

Reply:
Thx for the feedback. It's now fixed :-)